runtime/storage/sqldb/db.go

```diff
@@ -262,3 +261,17 @@ func (db *Database) Begin(ctx context.Context) (*Tx, error) {
 
 	return &Tx{mgr: db.mgr, txid: txid, std: tx}, nil
 }
+
+// Driver returns the underlying database driver for this database connection pool.
+//
+// It is currently of type [*pgxpool.Pool] and it's safe to immediately do a type conversion
+// to that type:
+//
+//	var db = sqldb.Named("mydatabase").Driver().(*pgxpool.Pool)
```

Comment:
I think the example should include a if guard
```suggestion
//	var db, ok = sqldb.Named("mydatabase").Driver().(*pgxpool.Pool)
//      if !ok {
//         panic("driver is not pgx")
/       }
```